variables.tf

```diff
@@ -33,3 +33,15 @@ variable "api_url" {
   description = "URL of alternative CAST AI API to be used during development or testing"
   default = "https://api.cast.ai/"
 }
+
+variable "subnets" {
+  type = list(string)
+  description = "Subnets for the cluster"
```

Comment:
Since subnets and security groups are optional it could be added to the description to make it more clear.

```
Optional custom subnets for the cluster. If not set subnets from the EKS cluster configuration are used.
```

```
Optional custom security groups for the cluster. If not set security groups from the EKS cluster configuration are used.
```